.github/workflows/app-cd.yml

```diff
@@ -6,8 +6,14 @@ permissions:
   contents: read  # This is required for actions/checkout
 
 # run on merge since you cannot push to main directly
+# on:
+#   push: 
+#     branches:
+#       - main
+#     paths:
+#       - front/**
```

Comment:
Commented code on main...

Reply:
yup! I was going to remove this after CR was approved because it's currently used to test CD on this PR